lib/src/widgets/movie_detail_information.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import '../../utils/constants.dart';
+import '../../utils/text_styles.dart';
+
+class MovieDetailInformation extends StatelessWidget {
+  final String date;
+  final String rating;
+
+  MovieDetailInformation({
+    required this.date,
+    required this.rating,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        vertical: Constants.paddingInformation,
+      ),
+      child: Column(
+        children: [
+          Text(
+            "Date: $date",
```

Comment:
You can add this to String file like movieDetailDateText = "Date: " and movieDetailRatingText = "Rating: "

Reply:
done